server/routes/handlers/outcomes/outcomesRouter.js

```diff
@@ -14,5 +15,6 @@ const outcomesRouter = express.Router({ mergeParams: true })
 outcomesRouter.get('/', defaults, outcomesMiddleware('NEW'), catchErrors(casesToResultHandler))
 outcomesRouter.get('/in-progress', defaults, outcomesMiddleware('IN_PROGRESS'), catchErrors(casesInProgressHandler))
 outcomesRouter.get('/resulted-cases', defaults, outcomesMiddleware('RESULTED'), catchErrors(getResultedCasesHandler))
+outcomesRouter.post('/hearing/:hearingId/outcome/assign', defaults, catchErrors(assignHearingOutcomeRouteHandler))
```

Comment:
```suggestion
outcomesRouter.post('/hearing/:hearingId/assign', defaults, catchErrors(assignHearingOutcomeRouteHandler))
```
?
outocme is already taken care in the  is already taken care of here https://github.com/ministryofjustice/prepare-a-case/blob/87e1c6833be200c89fe1976168c90d8b244d9f91/server/routes/index.js#L584